bin/add-cmderplugin.ps1

```diff
@@ -0,0 +1,161 @@
+<#
```

Comment:
Again, I don't think this should be installed into `bin`, because bin (the same as config) is under the user's control and therefore changes by the user should not be overwritten, but installing into bin would do this.

On the other hand, I don't see a better dir... The only way would be a `vendor/cmder` dir, which the user either needs to call the script by a absolute name or to add that dir to the path...